neuraldbg/__init__.py

```
@@ -241,8 +274,11 @@ def _classify_optimizer_health(self) -> OptimizerHealth:
 
     def _install_hooks(self):
         """Install forward and backward hooks to extract semantic events."""
+
         @dynamo_disable
-        def forward_hook(module: nn.Module, input: Tuple[torch.Tensor], output: torch.Tensor):
+        def forward_hook(
+            module: nn.Module, input: Tuple[torch.Tensor], output: torch.Tensor
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)